xmake/core/project/policy.lua

```diff
@@ -35,6 +35,8 @@ function policy.policies()
     if not policies then
         policies =
         {
+            -- enable clang std modulemap
+            ["c++.clang.module.stdmodules"]          = {description = "Enable clang std modulemap.", default = false, type = "boolean"},
```

Comment:
rename to `build.c++.clang.module.stdmodules` and move it below `build.c++.modules`
```
            ["build.c++.modules"]                = {description = "Enable C++ modules for C++ building.", type = "boolean"},
            ["build.c++.clang.module.stdmodules"]          = {description = "Enable clang std modulemap.", default = false, type = "boolean"},

```